pkg/cmd/init.go

```diff
@@ -251,6 +293,35 @@ func GetHelmRepositoryList() []*repo.Entry {
 	return nil
 }
 
+func GetHelmRelease() ([]*release.Release, error) {
+	actionConfig := new(action.Configuration)
+	client := action.NewList(actionConfig)
+
+	if err := actionConfig.Init(settings.RESTClientGetter(), "", os.Getenv("HELM_DRIVER"), info); err != nil {
```

Comment:
what will happen if `os.Getenv("HELM_DRIVER")` is empty?

Reply:
Init will create a new secret in k8s to store release info.